### ISSUE TITLE
Meals Prepared does not include Oven made, Fried, Grilled, Ice Creamed, Snackbarred, Pastries, or Cereal items.  

### ISSUE THREAD

**Issue by SGuivenne** [d3athrow/vgstation13#16730]
#### Description of issue

Anything put into what I THINK is the goofstation cooking system stuff (ovens, grills, cereal, icecream, candy) does not count towards meals made at round end.  

#### Difference between expected and actual behavior

Expected these items to count, they did not.

#### Steps to reproduce 

Cook a shit ton of things in systems other than the microwave, see round end stats.

#### Specific information for locating

Probably effects all non-microwave cooking or later additions

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [ ] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [ ] Issue happened in a recent (less than 7 days ago) round
- [ ] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

**Comment by yclatious** [comment]
You didnt cook dick, you shoved random garbage into a machine that made crap for you, the lowest effort foodjunk possible.

Go make actual food instead of being a shit chef if you want to be proud of a big number of made meals, your stuff isnt worth making into kibble, much less fed to anyone.

**Comment by SGuivenne** [comment]
RUDE

**Comment by yclatious** [comment]
But true, now go whip the botanists and make some neat dishes, nobody wants ambrosia jaw crackers or wheat cereal.

**Comment by Cyadox** [comment]
It'd be too abusable. 
You need literally 0 ingredients to make ice cream, and all the other machines can make food out of practically anything.
If you really want to cheese that score just make pasta or some other shit that takes 5u to make.

**Comment by Exxion** [comment]
I mean... yes, you could get infinite points like that
But I'm not sure it really counts as abusable because the points don't *do* anything

**Comment by yclatious** [comment]
Do not enable or validate the Goof machine users exxoin, there are fates worse than basic burgers and pasta from every chef.

**Comment by Exxion** [comment]
Yeah, I'm not saying that just throwing garbage into an oven should count for what actually cooking something does
I just also don't think it actually matters, really at all

**Comment by yclatious** [comment]
Something something principle, id rather die starving or eating dans pies than eating ambrosia jawbreakers, eggplant cereal or burnt mess cake.

**Comment by Exxion** [comment]
I'd suggest just making them count for some small fraction of a meal, but then some fuck would get the highest-ever chef score by just cooking the same object over and over for the entire round, simply for the sake of being obnoxious

Also it has occurred to me that our avatars look rather similar

**Comment by EenKogNeeto** [comment]
Meals Prepared: 427 (2135 Points) is my best score as chef. I die inside when I see people making non meme food with the frier/oven/etc. Faggot Salad, Donk Pocket Donk Pocket, etc. I would be up for the system being more robust or clear on how it scores things, but really you can feed the entire crew with two vanilla cakes properly prepared.

**Comment by Blithering** [comment]
Deep fried donk pocket cereal isn't a meal by anyone's standards, maybe have a separate counter in the weird section for the number of goof foods made, perhaps with some tally for the largest number of ingredients in a single item
